go.mod

```
@@ -20,9 +20,11 @@ require (
 	github.com/gofrs/flock v0.8.1
 	github.com/hashicorp/hcl/v2 v2.10.0
 	github.com/influxdata/go-syslog/v3 v3.0.0
+	github.com/jmoiron/sqlx v1.2.1-0.20190826204134-d7d95172beb5
 	github.com/joho/godotenv v1.3.0
 	github.com/lucasb-eyer/go-colorful v1.2.0
 	github.com/mattn/go-isatty v0.0.13
+	github.com/mattn/go-sqlite3 v1.9.0
```

This version of go-sqlite is affected by this bug that's affecting me (in a very minor way):
https://github.com/mattn/go-sqlite3/issues/803

It's fixed in 1.14
```suggestion
	github.com/mattn/go-sqlite3 v1.14.8
```